docs/usage/key-concepts/scheduling.md

```diff
@@ -0,0 +1,134 @@
+
+By default, Renovate schedules use the UTC timezone.
+You can override the default timezone by setting your own `timezone` config option.
+
+## Global schedule vs specific schedule
+
+When we talk about scheduling Renovate, there are 2 senses in which you can schedule Renovate:
+
+- A global sense: when the bot is allowed to do work at all
+- A specific sense: when Renovate is allowed to look for updates to a specific dependency
+
+Because Renovate defaults to "always on" and "open PRs right away" it can be overwhelming your repository with notifications of new PRs.
+To reduce overwhelm, we provide scheduling tools.
+
+You may want to update certain repositories less often, or you may even want to use different schedules for specific packages.
+
+## Scheduling use cases
+
+Some common reasons to schedule when Renovate runs:
+
+- Make Renovate run outside office hours, to free up continuous integration resources for developers during the day
+- Get updates for certain packages on a regular interval instead of right away
+- Reduce Renovate bot PR notifications during the day
+
+## Customizing the schedule
+
+You can customize when Renovate runs, by using the `timezone` and `schedule` configuration options.
+
+At a high level you need to follow these steps:
+
+1. Tell Renovate what `timezone` you want to use
+1. Learn about the scheduling syntax
+1. Optional: limit when Renovate bot itself can run, via custom config or _schedule preset_
+1. Optional: create packageRules with a custom `schedule` for specific packages
+
+### Setting your timezone
+
+By default Renovate schedules use the UTC timezone.
+If you want Renovate to use your local time, use the `timezone` configuration option.
+The timezone must be a valid [IANA time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+
+You can set a specific time zone in your local config file like this:
+
+```json
+{
+  "timezone": "America/Los_Angeles"
+}
+```
+
+Read our docs on the [timezone](https://docs.renovatebot.com/configuration-options/#timezone) configuration option.
+
+### Scheduling syntax
+
+After you've set your local timezone, you can define days of week or hours of the day in which Renovate is allowed to make changes.
+
+Examples of the kind of schedules you can create include:
+
+```
+every weekend
+before 5:00am
+[after 10pm, before 5:00am]
+[after 10pm every weekday, before 5am every weekday]
+on friday and saturday
+```
+
+**Warning:** Renovate does not support scheduled minutes or "at an exact time" granularity.
+
+Renovate uses the [@breejs/later](https://github.com/breejs/later) library to parse the text.
+Read the parser documentation at [breejs.github.io/later/parsers.html#text](https://breejs.github.io/later/parsers.html#text) for more details.
+The _@breejs/later_ library also handles the concepts of "days", time_before", and "time_after".
+
+### Limit when Renovate bot itself runs
```

Comment:
The below section is not about limiting when Renovate runs - it's more examples of in-repo config.

Renovate's schedule is determined by the bot admin using tools such as `cron`.